examples/fluid.amp.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8">
+    <title>DoubleClick with Fluid Example</title>
+    <link rel="canonical" href="http://nonblocking.io/" >
```

Comment:
Nit: trailing space unnecessary. Would be better to use single-quotes throughout for consistency.

Reply:
Trailing space removed. Not sure about the single-quotes, though. Looking at other examples in the directory, it seems like most use double-quotes.

Reply:
Ok. As long as we're consistent.